### DISCUSSION TITLE
checking for changes in the body models in the new version

### DISCUSSION THREAD

**Discussion by YaraAmin** [vladmandic/human#187] [Q&A] [answered]
Hi
did you change anything in the body model/output in the new version? sometimes it can't detect body even if there's a body in the frame?

**Comment by vladmandic** [comment]
Default model is the same ('movenet-lightning), but I completely redesigned **detector** caching  
for **face**, **body **and **hand** modules  

Can you confirm this is the cause? Set `config.cacheSensitivity = 0` to disable caching  

I planned to keep 2.3 version in beta for a while longer while this is ironed out, but made a decision to release ahead of time due to Chrome GPU memory leak issue  

If you can find a good reproduction, I'd be happy to look at it

**Reply by vladmandic** [reply]
I just posted a major update on git if you want to take a look. Still needs polishing before publishing a new version, but should be much better.

**Reply by YaraAmin** [reply]
I'm already setting config.cacheSensitivity to 0
I will try the new update. thanks!

**Comment by vladmandic** [comment, marked as answer]
2.3.2 is released, no breaking changes but a lot of improvements to body and hand detections under the hood.
